### ISSUE TITLE
Unable to build testsuite

### ISSUE THREAD

**Issue by kbobyrev** [GothenburgBitFactory/taskwarrior#1979]
I don't have much experience with building Taskwarrior, but it seems that there is some issue I've run into while trying to build the testsuite.

* What I'm doing is `$ make test all`
* I expect the build to finish correctly
* I get the `undefined reference` error

I'm using Arch Linux, using both GCC 7.3.0 and LLVM Clang 5.0.1 does not result in a successful build.

Here's the [log](https://github.com/GothenburgBitFactory/taskwarrior/files/1790620/taskwarrior.txt).

**Comment by pbeckingham** [comment]
This works for me, can you confirm the commands you used?

```
$ git clone --recursive -b 2.6.0 https://github.com/GothenburgBitFactory/taskwarrior tw-1979.git
$ cd tw-1979.git
$ cmake .
$ make
$ cd test
$ make
```

**Comment by omtcvxyz** [comment]
@pbeckingham Yes, executing the exact sequence of commands you provided yields an error which I described before. The problem occurs on the last `$ make` step when in `test` subdirectory.

**Comment by pbeckingham** [comment]
Interesting. I cannot replicate this, but then I don't have your setup.  I'll retry on Arch.

**Comment by omtcvxyz** [comment]
Yes, honestly it is pretty weird because I see the `task` library being linked in the tests compilation scripts. Also, `Context::getContext()` is not only used in `col/` sources. That's why I didn't find out why this is an issue yet.

**Comment by omtcvxyz** [comment]
Okay, so as far as my understanding goes the following creates an issue: before mentioned commit, there was `Context context` declaration in each test source file, now there is none. Adding it back is a working solution, although I am not sure it is the best one.

**Comment by pbeckingham** [comment]
When our CI system is back online, I will fix this.  Soon.

**Comment by tbabej** [comment]
This was fixed in 9a8890f47978aaffaab0a6882a0a0f97c81fdbf8.
